### PR TITLE
Updated the interface for admission controller

### PR DESCRIPTION
**Pull request by ravisantoshgudimetla**
Updated the documentation to have latest admission controller interface.

### CODE REVIEW

File: contributors/design-proposals/admission_control.md [hunk 1]
```diff
@@ -44,20 +44,41 @@ An **AdmissionControl** plug-in is an implementation of the following interface:
...
 type Interface interface {
-  // Admit makes an admission decision based on the request attributes
-  // An error is returned if it denies the request.
-  Admit(a Attributes) (err error)
+	// Admit makes an admission decision based on the request attributes
+	Admit(a Attributes) (err error)
+
+	// Handles returns true if this admission controller can handle the given operation
+	// where operation can be one of CREATE, UPDATE, DELETE, or CONNECT
+	Handles(operation Operation) bool
```
Comment (bsalamat) [inline contributors/design-proposals/admission_control.md]: I think "Admissible" would be a better name for the function.

Comment (deads2k) [inline contributors/design-proposals/admission_control.md]: > I think "Admissible" would be a better name for the function.

This looks to be sync-ing the actual impl, not suggesting any changes.

Comment (ravisantoshgudimetla) [inline contributors/design-proposals/admission_control.md]: Thanks @bsalamat, @deads2k for reviewing- The only question that I have is should we include how to enable it in local-cluster script to test locally

**Comment by derekwaynecarr** [comment]
Hmm, rather than let this get out of sync, I think the interface is stable enough that we could just point to the file at some point...

**Comment by derekwaynecarr** [comment]
I guess the main point is this is a design-proposal document (from the earliest days of kubernetes) and not a user guide.  So explaining things like local clusters, etc. feels more tilted to user guide stuff.

**Comment by ravisantoshgudimetla** [comment]
>Hmm, rather than let this get out of sync, I think the interface is stable enough that we could just point to the file at some point...

Makes sense. I can make that change.

>I  guess the main point is this is a design-proposal document (from the earliest days of kubernetes) and not a user guide. So explaining things like local clusters, etc. feels more tilted to user guide stuff.

The incoming links to page on github are from developer docs, if this is a design doc, we should remove incoming links from developer docs or have a new doc which talks about how to add adm controller. The other day when I was writing custom admission controller, I had to look at older PR's to know which files are to be touched for testing.

**Comment by derekwaynecarr** [comment]
@ravisantoshgudimetla -- good point, there is a doc that states what admission controllers to enable/disable and in what order.  let me see if i can locate that.

**Pull request closed without merging**